onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -1861,6 +1861,25 @@ TEST(ReductionOpTest, ArgMin) {
   test.Run();
 }
 
+TEST(ReductionOpTest, ArgMin_Double_Type) {
+  OpTester test("ArgMin", 11);
+  test.AddAttribute("axis", (int64_t)0);
+  test.AddAttribute("keepdims", (int64_t)1);
+  test.AddInput<double>("data", {3, 2, 2},
+                       {1.0, 2.0,
+                        3.0, 4.0,
+
+                        5.0, 6.0,
+                        7.0, 8.0,
+
+                        9.0, 10.0,
+                        11.0, 12.0});
+  test.AddOutput<int64_t>("reduced", {1, 2, 2},
+                          {0, 0,
+                           0, 0});
+  test.Run();
```

Comment:
nit: I realize this is a cut-and-paste of the existing test but would be slighter nicer in general to have a test where all the results aren't all the same value, as that could mask an issue.

Reply:
Ok, i'll come up with an example where float and double don't return the same results.